apps/qubit/templates/_header.php

```diff
@@ -38,14 +46,6 @@
 
   </nav>
 
-  <div id="search-bar">
-
-    <?php echo get_component('menu', 'browseMenu', ['sf_cache_key' => $sf_user->getCulture().$sf_user->getUserID()]); ?>
-
-    <?php echo get_component('search', 'box'); ?>
-
-  </div>
-
```

Comment:
Most custom themes won't be able to benefit from this solution. While B2 is being deprecated and themes should be updated, it would be great if we could find another solution. Don't worry too much if there is no "easy" alternative.

Reply:
Also, check arArchivesCanadaPlugin.

Reply:
@jraddaoui just updated this. Unfortunately there isn't an easy and clean alternative. The arArchiveCanadaPlugin ended up needing a small styling change for this to work, which also seems a little awkward given what the change is for.